src/nodes/importConfigurationCoordinates.cpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2025 Team Dissolve and contributors
+
+#include "nodes/importConfigurationCoordinates.h"
+#include "nodes/dissolve.h"
+
+ImportConfigurationCoordinatesNode::ImportConfigurationCoordinatesNode(Graph *parentGraph) : Node(parentGraph)
+{
+    addInput<Configuration *>("Configuration", "Configuration to which coordinates will be imported", configuration_);
+    addOption<std::string>("FilePath", "File path", filePath_);
+    addOption<CoordinateImportFileFormat::CoordinateImportFormat>("FileFormat", "File format", format_);
+    addOutput<Configuration *>("Configuration", "Output configuration", configuration_);
```

Comment:
Just a bit of formatting for consistency with other changes I'm adding in.
```suggestion
    // Inputs
    addInput<Configuration *>("Configuration", "Configuration to which coordinates will be imported", configuration_);

    // Options    
    addOption<std::string>("FilePath", "File path", filePath_);
    addOption<CoordinateImportFileFormat::CoordinateImportFormat>("FileFormat", "File format", format_);

    // Outputs    
    addOutput<Configuration *>("Configuration", "Output configuration", configuration_);
```